org.cockpit_project.CockpitClient.metainfo.xml

```diff
@@ -18,6 +18,21 @@
     </p>
   </description>
 
+  <screenshots>
+    <screenshot>
+      <caption>Cockpit Client login screen</caption>
+      <image>http://cockpit-project.org/images/flatpak/flatpak-screenshot-login.png</image>
```

Comment:
Same as in https://github.com/cockpit-project/cockpit/pull/16646 -- https please, these are redirected anyway.

Reply:
fixed, thanks